modules/web/src/app/shared/components/label-form/component.ts

```diff
@@ -71,6 +71,7 @@ export class LabelFormComponent implements OnChanges, OnInit, OnDestroy, Control
   @Input() labelHintKey: string;
   @Input() inheritedLabels: object = {};
   @Input() noValidators = false;
+  @Input() hideRestrictedLabels = false;
```

Comment:
Are we setting `hideRestrictedLabels` to `true` somewhere? I couldn't find it in this PR.

Reply:
Nope, we are not. It's just there because this is something that will be useful for some upcoming changes that I have planned.

Reply:
Okay. In that case I think `*ngIf="!(hideRestrictedLabels && label.get('key').errors?.labelKeyNameRestricted)"` might be a bit wrong because if `hideRestrictedLabels` is `true` and there is `labelKeyNameRestricted` error on `key` then we won't be able to delete this label. Wdyt?

Reply:
Which is fine and intended.